### PR TITLE
application structure is specified

### PR DESCRIPTION
**Pull request by efremnikolaus**
_The initial structure was set to make it easier to work with the application in the future._
_Briefly about the application:_

1. **API-Gateway:** This component is the entry point for external queries.
2. **Discovery-Server:** This service is responsible for registering and detecting micro-services in the execution environment.
3. **Template-Service:** This is the service that manages the creation and storage of message templates for notifications.
4. **Recipient-Service:** This service manages information about notification recipients.
5. **File-Service:** This service is responsible for managing files in the context of notifications.
6. **Notification-Service:** This service is the central element of the system that processes the logic of creating and sending notifications.

### CODE REVIEW

File: file-service/src/main/resources/application.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+spring:
+  application:
+    name: file-service
+server:
+  port: 0
```
Comment (noharagg) [inline file-service/src/main/resources/application.yaml:5]: what does it mean , port : 0

File: notification-service/src/main/resources/application.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+spring:
+  application:
+    name: notification-service
+server:
+  port: 0
```
Comment (noharagg) [inline notification-service/src/main/resources/application.yaml:5]: same q

File: file-service/src/main/resources/application.yaml (hunk 1 above)
Comment (efremnikolaus) [reply file-service/src/main/resources/application.yaml:5]: This means that the port will be determined randomly.

File: notification-service/src/main/resources/application.yaml (hunk 2 above)
Comment (efremnikolaus) [reply notification-service/src/main/resources/application.yaml:5]: [answer](https://github.com/efremnikolaus/emergency-notification-system/pull/3#discussion_r1437830264)

**Review by noharagg — COMMENTED** [review]
LGTM

**Review by noharagg — APPROVED** [review]
LGTM

**Pull request merged**